org_status/org_status.py

```diff
@@ -199,7 +200,8 @@ def main():
         present_status(org_status, args.no_color)
 
     if args.export_repos:
-        export_data = encode_repo_list(all_repositories, args.format, styled)
+        export_data = encode_repo_list(
```

Comment:
while this is nice, and should be kept, the objective of the issue is to sort the status results (the tools normal output) by the repo 's last commit date

Reply:
Ah ok, I misunderstood the intent of the original issue, my bad.